### PR TITLE
schutzbot: try to speed things up without losing functionality

### PR DESCRIPTION
**Pull request by teg**
Drop some (arguable) redundant boot tests and run other tests more in parallel.

### CODE REVIEW

**Comment by msehnout** [comment]
I restarted the CI as it failed after almost 13 hours :roll_eyes: ....

File: tools/test-case-generators/generate-test-cases [hunk 1]
```diff
@@ -94,6 +94,7 @@ def generate_test_case(test_type, distro, arch, output_format, test_case_request
             pass
     with open(file_name, 'w') as case_file:
         json.dump(test_case, case_file, indent=2)
+        case_file.write("\n")
```
Comment (msehnout) [inline tools/test-case-generators/generate-test-cases:97]:
```
 Fixing Whitespace, what else would you do at this time?
```
:-D

**Review by msehnout — APPROVED** [review]
Sounds reasonable to me.

File: tools/test-case-generators/generate-test-cases (hunk 1 above)
Comment (teg) [reply tools/test-case-generators/generate-test-cases:97]: Hehehe, I know I'm not the only one bothered by whitespace ;)

**Review by ondrejbudai — APPROVED** [review]
Fixes https://github.com/osbuild/osbuild-composer/issues/1201 !!!!!!!!!

**Comment by msehnout** [comment]
> Fixes #1201 !!!!!!!!!

We must save Ondřej's soul by merging this PR!

**Comment by teg** [comment]
> Fixes #1201 !!!!!!!!!

LOL, I even missed the issue!

**Pull request merged**